source/MarlinProcessors/src/DrawBeamCalFromDD4hep.cpp

```diff
@@ -314,9 +320,14 @@ void DrawBeamCalFromDD4hep::drawCartesianGridXY() {
 
 void DrawBeamCalFromDD4hep::drawPolarGridRPhi2() {
 
+  gStyle->SetOptStat(0);
+  gStyle->SetOptTitle(0);
+  gStyle->SetTitleSize(0.05, "xyz");
+  gStyle->SetLabelSize(0.05, "xyz");
+
   TCanvas c1("rphi2", "rphi2", 800, 800);
   TH2D dummy("dummy", "dummy", 20, 0, 20, 200, 0, 200); //16 bits to get colours from pal->GetValueColor
-  TH2D g("g", "g;x [cm]; y [cm]", 200, -20, 20, 200, -20, 20); //16 bits to get colours from pal->GetValueColor
+  TH2D g("g", "g;x [cm]; y [cm]; E_{dep} (GeV)", 200, -20, 20, 200, -20, 20); //16 bits to get colours from pal->GetValueColor
```

Comment:
Use brackets `[GeV]` like for `[cm]`

Reply:
Done. But I now realize that `E_{dep} [GeV]` is unnecessary as no z-axis is drawn. So the title never gets drawn either. Should I remove it?

Reply:
Leave it there, it is a good hint to know what the histogram contains, even if it is just in the code.